src/client.coffee

```diff
@@ -43,6 +44,9 @@ class SlackClient
     @rtm = new RtmClient options.token, options.rtm
     @web = new WebClient options.token, { maxRequestConcurrency: 1 }
 
+    unless isNaN(options.apiPageSize)
+      SlackClient.PAGE_SIZE = parseInt(options.apiPageSize, 10)
```

Comment:
I know this is a pretty easy to make this work. However, now that the value is no longer a constant, we may want to use an instance filed like `@page_size` over a class field plus replace the references to `SlackClient.PAGE_SIZE` in the code.

Reply:
Thanks, yeah I was thinking that too, needed someone to confirm my own doubt. I used @apiPageSize to follow existing camelCase naming convention.